tests/utils/datavolume.go

```diff
@@ -317,6 +317,30 @@ func NewDataVolumeWithHTTPImportToBlockPV(dataVolumeName string, size string, ht
 
 // NewDataVolumeWithExternalPopulation initializes a DataVolume struct meant to be externally populated
 func NewDataVolumeWithExternalPopulation(dataVolumeName, size, storageClassName string, volumeMode corev1.PersistentVolumeMode, dataSource, dataSourceRef *corev1.TypedLocalObjectReference) *cdiv1.DataVolume {
+	dataVolume := &cdiv1.DataVolume{
```

Comment:
Could you explain what happened here?
Usually, I thought we shift towards using the storage API mostly

Reply:
The storage api obscures the matching a bit, just have to be consistent either way (all pvc or all storage).  I took the PVC route to be explicit as possible.  Basically when I originally wrote the test I was using `pvc` to create the source then didn't get a match I used `storage` for target.

Reply:
gotcha